src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -195,8 +195,8 @@ public void SignUp_007_signUpWithFacebook() {
   public void SignUp_008_signupJapaneseUser() {
     SignUpPageObject signUp = new WikiBasePageObject(driver).navigateToSpecialSignUpPage(wikiURL);
     signUp.disableCaptcha();
-    String userName = "品質管理" + signUp.getTimeStamp();
-    String password = "品質管理管理" + signUp.getTimeStamp();
+    String userName = "ユーザー" + signUp.getTimeStamp();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "userName" which hides the field declared at line 36. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)